codegens/csharp-restsharp/lib/restsharp.js

```diff
@@ -29,6 +29,12 @@ function makeSnippet (request, options) {
     snippet += 'client.FollowRedirects = false;\n';
   }
   snippet += `var request = new RestRequest(${isUnSupportedMethod ? '' : ('Method.' + request.method)});\n`;
+  if (request.body && request.body.mode === 'graphql' && !request.headers.has('Content-Type')) {
```

Comment:
is request.headers case-insensitive?

Reply:
yes. content-type or Content-Type, both the cases it returns the same